.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
```

Comment:
v2 and not master here? I think we should be consistent and probably not worth the risk of tracking master for this?

Reply:
Error on my side